components/apimgt/org.wso2.carbon.apimgt.internal.service/src/main/resources/api.yaml

```
@@ -2168,7 +2247,132 @@ components:
+        revisionId:
+          type: string
+          description: Unique identifier of the API revision that was deployed or undeployed.
+          example: "f46c7962-cd36-4aa6-b804-8d66ad6a0dc4"
+        errorCode:
+          type: integer
+          description: A unique code identifying the error, present only if deploymentStatus is 'FAILURE'.
+        errorMessage:
+          type: string
+          description: A descriptive error message, present only if deploymentStatus is 'FAILURE'.
+    DeploymentAcknowledgmentResponse:
+        type: object
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)